HealthcareBase/Service/UsersService/PatientService/IPatientAccountService.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using HealthcareBase.Model.Users.Employee;
+using HealthcareBase.Model.Users.Employee.Doctors;
+using HealthcareBase.Model.Users.Patient;
+using HealthcareBase.Model.Users.UserAccounts;
+using HealthcareBase.Model.Users.UserFeedback;
+
+namespace HealthcareBase.Service.UsersService.PatientService
+{
+    public interface IPatientAccountService
+    {
+        public PatientAccount CreateAccount(PatientAccount patientAccount);
+        public void DeleteAccount(PatientAccount patientAccount);
+        public PatientAccount GetAccount(Patient patient);
+        public PatientAccount GetAccount(int patientId);
+        public PatientAccount ChangePassword(PatientAccount account, string newPassword);
+        public PatientAccount AddFavouriteDoctor(Doctor doctor, PatientAccount account);
+        public PatientAccount RemoveFavoriteDoctor(Doctor doctor, PatientAccount account);
```

Comment:
It may be better to do these two functions through the doctor and patient identifiers

Reply:
Those are unnecessary brought from sims project.
They well be deleted sooner or later.